src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -2962,6 +2988,8 @@ private void cleanup(
                 private StringFW addressFrom;
                 private StringFW addressTo;
 
+                private boolean detachError;
```

Comment:
Suggest changing the type to `AmqpErrorType` instead of a boolean flag, may be more helpful in debugging.